marketplace/collect.schema.json

```diff
@@ -1,8 +1,6 @@
 {
-    "$id": "https://schemas.aramis.group/marketplace/collect.schema.json",
-    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
```

Comment:
Is it `http` or `https`? Why have you changed it here but not in the other files?